packages/@expo/cli/src/utils/telemetry/getContext.ts

```diff
@@ -11,8 +13,29 @@ export function getContext() {
   const platform = PLATFORM_TO_TELEMETRY_PLATFORM[os.platform()] || os.platform();
   return {
     os: { name: platform, version: os.release() },
```

Comment:
One suggestion: we could just use `os.platform()` as the name instead of mapping it to `Mac/Windows/Linux`. It doesn't change much for us since we could map it on the analytics side instead.

Reply:
<img width="403" alt="image" src="https://github.com/expo/expo/assets/1203991/0ee7227f-4aa8-463b-8737-c083f1f3fb36">